docker-compose.yml

```diff
@@ -73,11 +73,15 @@ services:
       context: .
       dockerfile: ./docker/Dockerfile
     restart: 'on-failure'
+    environment:
+      - PERIPHERAL_API_URL=http://sce-peripheral-api:8081/peripheralapi
     stdin_open: true
     links:
       - mongodb
     depends_on:
       - mongodb
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
undo the changes in this file, its wrong anyway, see https://github.com/SCE-Development/Clark/blob/9f0aee7de664b9de37b10b7c0228a2add453f60d/docker/Dockerfile#L16

Reply:
yes